docs/src/app/pages/BluetoothLEIntro.md

```diff
@@ -53,226 +89,740 @@ Bluetooth Low Energy, also referred to as Bluetooth LE or simply BLE, is a new p
 
 ![call BluetoothLE1 WriteFloatsserviceUuidcharacteristicUuidshortFloatvalues](blocks/BluetoothLE.WriteFloats.svg)
 
-+ <a name="WriteFloatsWithResponse"></a>`WriteFloatsWithResponse` – Writes one or more 8-bit integer values to a connected BluetoothLE device and wait for an acknowledgement via the FloatValuesWritten event. Service Unique ID and Characteristic Unique ID are required. The values parameter can either be a single numeric value or a list of values. If shortFloat is true, the values will be compressed to 16-bits. If shortFloat is false, the values are sent as 32-bit.
++ <a name="WriteFloatsWithResponse"></a>`WriteFloatsWithResponse` – Writes one or more IEEE 754 floating point values to a connected BluetoothLE device and waits for an
+ acknowledgement via the <a href="#FloatsWritten"><code>FloatsWritten</code></a> event.
+ Service Unique ID and Characteristic Unique ID are required. The values parameter can either
+ be a single numeric value or a list of values. If <code>shortFloats</code> is false,
+ then each numeric value will be sent as a 32-bit single precision floating point value.
+
+ __Parameters__:
+
+     * <code>serviceUuid</code> (<a href="http://appinventor.mit.edu/explore/ai2/support/blocks/text.html#string">_text_</a>) &mdash;
+       The unique identifier of the service passed in the read or register call.
+     * <code>characteristicUuid</code> (<a href="http://appinventor.mit.edu/explore/ai2/support/blocks/text.html#string">_text_</a>) &mdash;
+       The unique identifier of the characteristic in the read or register call.
+     * <code>shortFloats</code> (<a href="http://appinventor.mit.edu/explore/ai2/support/blocks/logic.html#true" target="_blank">_boolean_</a>)
+       Interpret the floats as 16-bit half-precision (true) or 32-bit single-precision (false).
+     * <code>values</code> (<a href="http://appinventor.mit.edu/explore/ai2/support/blocks/lists.html#makealist">_list_</a>) &mdash;
+       A list of values to write to the device.
 
 ![call BluetoothLE1 WriteFloatsWithResponseserviceUuidcharacteristicUuidshortFloatvalues](blocks/BluetoothLE.WriteFloatsWithResponse.svg)
 
-+ <a name="WriteIntegers"></a>`WriteIntegers` – Writes one or more 32-bit integer values to a connected BluetoothLE device. Service Unique ID and Characteristic Unique ID are required. The values parameter can either be a single numeric value or a list of values. If signed is true, the acceptable values are between -2147483648 and 2147483647. If signed is false, the acceptable values are between 0 and 4294967295.
++ <a name="WriteIntegers"></a>`WriteIntegers` – Writes one or more 32-bit integer values to a connected BluetoothLE device. Service Unique ID
+ and Characteristic Unique ID are required. The values parameter can either be a single numeric
+ value or a list of values. If signed is true, the acceptable values are between -2147483648 and
+ 2147483647. If signed is false, the acceptable values are between 0 and 4294967295.
```

Comment:
rendering as the beginning of a numbered list. Remove line break and it should be ok, I hope.